lib/rmt/rpm.rb

```diff
@@ -0,0 +1,10 @@
+module RMT
+  module Rpm
+  end
+end
+
+require_relative 'rpm/file_entry'
```

Comment:
Hmm, for better modularity. All RPM stuff is required in RPM module. You can require just the downloader without any of the RPM stuff.